templates/test/ci/prow-ipv6/kustomization.yaml

```diff
@@ -3,8 +3,18 @@ kind: Kustomization
 namespace: default
 resources:
   - ../../../flavors/ipv6
-  - cni-resource-set-ipv6.yaml
+  - ../prow/cni-resource-set.yaml
 patchesStrategicMerge:
   - ../patches/tags.yaml
   - ../patches/controller-manager.yaml
   - ../patches/cluster-cni.yaml
+configMapGenerator:
+  - name: cni-${CLUSTER_NAME}-calico
```

Comment:
```suggestion
  - name: cni-${CLUSTER_NAME}-calico-ipv6
```

Reply:
it's using ../prow/cni-resource-set.yaml so the configMap name is actually the same, it's just the content that's changing